code/modules/materials/Mat_Materials.dm

```diff
@@ -1440,6 +1440,18 @@ ABSTRACT_TYPE(/datum/material/fabric)
 		setProperty("thermal", 9)
 		setProperty("electrical", 7)
 
+/datum/material/censorium
```

Comment:
should probably be a child of one of the abstract parents like `/datum/material/metal` or /datum/material/organic`

Reply:
got it